server/data/licenceClient.ts

```diff
@@ -134,7 +134,7 @@ export class LicenceClient {
   async getDeliusIds(nomisUserName): Promise<DeliusIds[]> {
     const query = {
       text:
-        'select staff_id "staffCode", delius_username "deliusUsername" from v_staff_ids where upper(nomis_id) = upper($1)',
+        'select staff_id "staffIdentifier", delius_username "deliusUsername" from v_staff_ids where upper(nomis_id) = upper($1)',
```

Comment:
This looks like its reading the column that stores staff code as staffIdentifier? 
Isn't there a new column to read from?

Reply:
Should this be `staff_identifier "staffIdentifier"` ?

Reply:
Doh.  Fixed now.